src/packages/@common/utils/init-environment-tab.ts

```diff
@@ -8,23 +9,57 @@ class InitEnvironmentTab {
    * @param _workspaceId - Workspace mongo id to which Environment belongs to
    */
   constructor(_id: string, _workspaceId: string) {
+    // this._tab = {
+    //   id: _id,
+    //   tabId: uuidv4(),
+    //   name: "New Environment",
+    //   description: "",
+    //   variable: [
+    //     {
+    //       key: "",
+    //       value: "",
+    //       checked: true,
+    //     },
+    //   ],
+    //   isActive: false,
+    //   type: "LOCAL",
+    //   isSave: true,
+    //   isSaveInProgress: false,
+    //   workspaceId: _workspaceId,
+    //   timestamp: new Date().toString(),
```

Comment:
why this has been commented?

Reply:
It is the previous code which we are using while creating a new env  but now we have updated it because we have to create a new tab when we create a new env.